.buildkite/packaging.pipeline.yml

```diff
@@ -9,7 +9,7 @@ env:
   IMAGE_UBUNTU_X86_64: "family/platform-ingest-beats-ubuntu-2204"
```

Comment:
Can you test this pipeline manually? IIUC, you can trigger it:
- https://github.com/elastic/beats/blob/08e49732928bb9341212bfe8b8d667176f77455f/catalog-info.yaml#L1047-L1060

https://www.jvt.me/posts/2024/07/16/buildkite-trigger-fork/ could be a good docs for understanding how to use BK for forks 

Thanks

Reply:
https://github.com/elastic/beats/pull/44701#issuecomment-2976509047